pkg/ddevapp/config.go

```diff
@@ -711,17 +711,22 @@ func (app *DdevApp) CheckCustomConfig() {
 		}
 	}
 
-	customDockerPath := filepath.Join(ddevDir, "web-build")
-	if _, err := os.Stat(customDockerPath); err == nil {
-		dockerFiles, err := filepath.Glob(filepath.Join(customDockerPath, "*Dockerfile*"))
-		util.CheckErr(err)
-		dockerFiles = slices.DeleteFunc(dockerFiles, func(s string) bool {
-			return strings.HasSuffix(s, ".example")
-		})
-		if len(dockerFiles) > 0 {
-			printableFiles, _ := util.ArrayToReadableOutput(dockerFiles)
-			util.Warning("Using custom web-entrypoint.d configuration: %v", printableFiles)
-			customConfig = true
+	for _, buildType := range []string{"web-build", "db-build"} {
+		customDockerPath := filepath.Join(ddevDir, buildType)
+		if _, err := os.Stat(customDockerPath); err == nil {
+			dockerFiles, err := filepath.Glob(filepath.Join(customDockerPath, "Dockerfile*"))
```

Comment:
It wasn't clear to me why you couldn't glob the Dockerfile* and pre.Dockerfile* in one statement here.

Reply:
Because if you name some file `foo.Dockerfile`, it will not be added to `.ddev/.webimageBuild/Dockerfile`, but will be shown here, which is wrong.